### ISSUE TITLE
Cinnamon needs a decent Messaging Menu Applet

### ISSUE THREAD

**Issue by A-Shahbazi** [linuxmint/Cinnamon#4815]
The messaging menu applets available till now([+](http://cinnamon-spices.linuxmint.com/applets/view/24), [+](http://cinnamon-spices.linuxmint.com/applets/view/173), [+](http://cinnamon-spices.linuxmint.com/applets/view/212)) were mere launching menus(they just add available messaging/email client apps and their relevant launching commands to menu). They weren't so-called interactive. Below is an image describes what I mean by interactive(This image is from indicator-messages):
![messaging menu](https://cloud.githubusercontent.com/assets/7456141/11060927/b4410132-87b8-11e5-8c70-ab8491cf5a87.png)
1. Icon changes when there are new messages.
2. An indicator shown when an application is running.
When there are new messages menu items should be added to those default launching options(i.e. compose new message, contacts). These items should contain following:
3. Folder in which the new message is in e.g. Inbox.
4. Account which the new message belongs to e.g. myemailaddress@gmail.com
5. New messages count

**Comment by mfreeman72** [comment]
I completely agree! I love Cinnamon, but this is the one feature I've been desperately missing for the last several years! The Messaging Menu applets available on Cinnamon's Spices/Applets site are ok, but as the OP said, they're not interactive or informative at all. They're just links. I'd like something that actively notifies me of new e-mails, interacts with the software I have installed, etc. I'd also love it if IM software notifications (Pidgin, Skype, Empathy, etc.) could be included as well, like Ubuntu's Messaging Indicator Applet does.
